cucumber-rails.gemspec

```diff
@@ -27,13 +29,14 @@ Gem::Specification.new do |s|
   s.add_development_dependency('rspec', '~> 3.6')
   s.add_development_dependency('rubocop', '~> 0.66.0')
   s.add_development_dependency('rubocop-rspec', '~> 1.32.0')
+  s.add_development_dependency('sqlite3', '~> 1.3')
```

Comment:
This "may" cause issues with appraisals, but I know without this you can't just run a standard thing on your own.

It's a bit of chicken and egg scenario I admit. Hopefully nothing bad happens.